packages/uniforms-bootstrap3/src/components/fields/SelectField.js

```diff
@@ -64,4 +64,3 @@ const Select = props =>
 ;
 
 export default connectField(Select);
-
```

Comment:
our OS must handle it differently, I am seeing the newline in my editor and the line count on Github indicates the newline is there